src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -123,15 +127,20 @@ public void readyArmForNewNote() {
   }
 
   public void intake(double speed) {
-    rightShootMotor.set(speed);
-    leftShootMotor.set(speed);
+    rightShootMotor.setVoltage(speed);
+    leftShootMotor.setVoltage(speed);
   }
 
   public void stopMotors() {
     rightShootMotor.setControl(brake);
     leftShootMotor.setControl(brake);
   }
 
+  public void setVolts(double volts) {
+    rightShootMotor.setControl(new VoltageOut(volts));
+    leftShootMotor.setControl(new VoltageOut(volts));
+  }
+
```

Comment:
this entire method is unnesserary as it
1. causes build errors due to incorrect types
2. is already fixed and merged into main for some reason